### DISCUSSION TITLE
Preloader (Webflow)

### DISCUSSION THREAD

**Discussion by iOstap** [darkroomengineering/lenis#192] [Q&A]
Please tell me how to disable page scrolling for a certain time while the preloader is loading?
I used which blocked page scrolling when element with attribute was in view. but with the use of Lenis, this script stopped working.
I think it is possible to apply the attribute “data-lenis-stop” not by clicking on the element, but when the element is on the screen.
but I do not understand what needs to be added in the code to implement this attribute

**Comment by iOstap** [comment]
```
$(document).ready(function() {
  lenis.stop();

  setTimeout(function() {
    lenis.start();
  }, 4500);
})
```

**Comment by Lizkaned** [comment]
Have the same problem(, maybe you have already fixed this ?
